bindings/matrix-sdk-ffi/src/notification_settings.rs

```diff
@@ -375,7 +395,7 @@ impl NotificationSettings {
                 let parsed_room_idom_id = RoomId::parse(&room_id)
```

Comment:
preexisting: typo in variable name, `parsed_room_id`

Reply:
fixed